lab2.js

```diff
@@ -135,12 +183,42 @@ function variablify(string) {
   // you might want to use these string methods:
   //  split(), charAt(), toUpperCase()
   // and this array method: join()
+  var splitString = string.split(' ');//'one' 'two' 'three'
+
+  var currentWord = null;
+  var oldLetter = null;
+  var newLetter = null;
+  var newString = [];
+
+  for (var i = 0; i < splitString.length; i++) {
+
+    if (i === 0) {
+
+      currentWord = splitString[i];// 'one'
+      newString.push(currentWord);
```

Comment:
nice. might also want to figure out how to handle the case where you get a capitalized word as the first word.